puppetboard/static/css/puppetboard.css

```diff
@@ -44,15 +44,15 @@ h1.ui.header.no-margin-bottom {
   color: #AA4643;
 }
 
-.ui.label.failed {
+.ui.label.failed, .ui.label.events.failure {
```

Comment:
What happens on labels we don't know about so something like events.foobar (if foobar was in events list).